openapi/components/schemas/GatewayAccountConfig/PayEcards.yaml

```
@@ -0,0 +1,26 @@
+description: PayEcards gateway configuration.
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: PayEcards credentials object.
+        properties:
+          clientKey:
+            type: string
+            description: Client Key of the PayEcards gateway.
```

nitpick: capitalization 
```suggestion
            description: Client key of the PayEcards gateway.
```